app/models/calagator/source/parser/facebook.rb

```diff
@@ -21,11 +24,17 @@ class Source::Parser::Facebook < Source::Parser
     }
 
   def to_events
+    return [] unless Calagator.facebook_access_token.present?
```

Comment:
Yeah, it really should. I went back and forth on it when implementing it originally, but I think that handling it with an exception here would be better.